src/Model/PluginFamily.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+namespace WPMedia\PluginFamily\Model;
+
+/**
+ * Handles the data to be passed to the frontend.
+ */
+class PluginFamily {
+	/**
+	 * Get filtered plugins.
+	 *
+	 * @param string $main_plugin Main plugin installed.
+	 *
+	 * @return array
+	 */
+	public function get_filtered_plugins( string $main_plugin ): array {
+		$plugins = require_once 'wp_media_plugins.php';
```

Comment:
I'm not sure I see the value there compared to a get method in this class that would return the array

Reply:
Sounds good